File: WaveHomework/WaveHomework/Models/Expense.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WaveHomework.Models
{
    public class Expense
    {
        public int Id { get; set; }
        public DateTime Date { get; set; }
        public string Category { get; set; }
        public string EmployeeName { get; set; }
        public string EmployeeAddress { get; set; }
        public string ExpenseDescription { get; set; }
        public decimal PretaxAmount { get; set; }
        public string TaxName { get; set; }
        public decimal TaxAmount { get; set; }
    }
}

File: models/expense.js
'use strict';

module.exports = function(sequelize, DataTypes) {
  var Expense = sequelize.define('Expense', {
    date: DataTypes.DATE,
    category: DataTypes.STRING,
    employee_name: DataTypes.STRING,
    employee_address: DataTypes.STRING,
    expense_description: DataTypes.STRING,
    pretax_amount: DataTypes.DOUBLE,
    tax_name: DataTypes.STRING,
    tax_amount: DataTypes.DOUBLE
  }, {
    classMethods: {
      associate: function(models) {
        // associations can be defined here
      }
    }
  });
  return Expense;
};
